### DISCUSSION TITLE
[Feature] Facial recognition on video

### DISCUSSION THREAD

**Discussion by DaanHend** [immich-app/immich#7953] [Feature Request]
### The feature

Immich offers a great facial recognition functionality. This is now based on the thumbnails of photos and on video's.

I would like to see facial recognition also be based on a video. I'm not familiar with thetechniques to accomplish. But I can understand that it will be quite intensive for the host machine. As a video is just a bunch of pictures.
So that would mean that it could be behind an option to enable. But it would be a cool addition.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #5936

**Reply by DaanHend** [reply]
How did you find it so quick? I couldnt find anyhing about 👍

**Reply by alextran1502** [reply]
The man has all the discussion topics segmented and allocated in his head 😄

**Reply by bo0tzz** [reply]
`is:open category:"Feature Request" face video` 😛
